test/unit_tests/apps/financial-well-being/js/fwb-results-spec.js

```diff
@@ -124,8 +124,8 @@ describe( 'fwb-results', () => {
   } );
 
   it( 'initialize the expandables on page load', () => {
-    expect( expandableTarget.getAttribute( 'aria-pressed' ) ).toBe( 'false' );
-    expect( expandableContent.getAttribute( 'aria-expanded' ) ).toBe( 'false' );
+    // expect( expandableTarget.getAttribute( 'aria-pressed' ) ).toBe( 'false' );
```

Comment:
If these aren't passing but are expected to in the future, changing the test to `xit` would be better.

Reply:
Aria pressed doesn't exist in CF expandables, but it probably should. I'll convert them to silenced.